clusterloader2/pkg/prometheus/manifests/exporters/windows_node_exporter/windows-exporter-daemonset.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: windows-exporter
+  name: windows-exporter
+  namespace: monitoring
+spec:
+  selector:
+    matchLabels:
+      app: windows-exporter
+  template:
+    metadata:
+      labels:
+        app: windows-exporter
+    spec:
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\system"
+      hostNetwork: true
+      initContainers:
+        - name: configure-firewall
+          image: mcr.microsoft.com/windows/nanoserver:1809
```

Comment:
this will work on WS 2019/2022 since this is a hostprocess container but in the future we should make this a multi arch image to save some space (the image gets pulled but is not actually used so on WS2022 we would have to pull the full image where as on 2019 the base layer is probably already pulled)